terminal.go

```diff
@@ -238,6 +238,7 @@ func New() *LightningTerminal {
 type stores struct {
 	accounts accounts.Store
 	sessions session.Store
+	rules    firewalldb.RulesDB
 
 	firewall     *firewalldb.DB
```

Comment:
there is some redundancy between rules and firewall, is it possible to remove one of them, or how will that look in the end? will we have a single firewall db here or the individual RulesDB/PrivacyDB?

Reply:
right, I think it's still possible and rules it not needed, unless I misunderstood you, see the patch
[changes.txt](https://github.com/user-attachments/files/19758562/changes.txt)

Reply:
ah ok yes indeed - you're totally right. Thanks for the call out 🙏

Reply:
thank you for the update!

Reply:
ah, the diff needed one more thing i think. fixed now i hope 🤞